app/models/location.rb

```diff
@@ -5,7 +5,7 @@ def should_generate_new_friendly_id?
     slug.blank? || name_changed?
   end
 
-  has_many :events
+  has_many :events, class_name: "::Event"
```

Comment:
really shouldn't need to specify this... were you running into something here?

Reply:
I was getting an ActiveStorage error and found that as a fix, which worked in the moment. Now that I removed it the tests are still passing so maybe something else I subsequently changed resolved the same issue.